migrations/upgrade-3-0-0.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * Apply any database upgrades required for 3.0.0.
+ *
+ * Autoresponder configuration was moved from admin settings to widget settings.
+ * All widgets must be updated with an autoresponder field.
+ *
+ * @since 3.0.0
+ */
+function smailyforwp_upgrade_3_0_0() {
```

Comment:
Even better would be to return anonymous function assigned as a variable and call it when running upgrades. That way you do not create lingering function definitions.

For example:
```php
$up = function() {
    // ...
};
```

And then you could do inside migrations:
```php
$up = NULL;
require 'migration.php';
if ( is_callable( $up ) ) {
    $up();
}
```

Reply:
Added